dandi/consts.py

```diff
@@ -134,3 +134,6 @@
 #: HTTP response status codes that should always be retried (until we run out
 #: of retries)
 RETRY_STATUSES = (500, 502, 503, 504)
+
+external_file_extensions = [".mp4", ".avi", ".wmv", ".mov", ".flv"]
+external_file_modules = ["processing", "acquisition"]
```

Comment:
This is only used once in the code.  Is there a need to define it in `consts.py`?

Reply:
we might want to use it again for other assisting datatypes that DANDI could host in the future. This just defines where in the nwbfile we want to restrict the lookup for external_files.